samples/graph-pinned-messages/nodejs/teamsAppManifest/manifest.json

```diff
@@ -0,0 +1,47 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
```

Comment:
Did we deploy this sample?

![image](https://user-images.githubusercontent.com/59052547/182854166-20a6b38f-0097-4897-a8a3-41522e67846a.png)